tests/api.c

```diff
@@ -1346,6 +1346,63 @@ static int test_wolfSSL_CTX_load_verify_locations(void)
     return 0;
 }
 
+static int test_wolfSSL_CTX_load_system_CA_certs(void)
+{
+    int ret = 0;
+
+#if !defined(NO_FILESYSTEM) && !defined(NO_CERTS) && !defined(NO_WOLFSSL_CLIENT)
+    WOLFSSL_CTX* ctx;
+
+    ctx = wolfSSL_CTX_new(wolfSSLv23_client_method());
+    if (ctx == NULL) {
+        ret = -1;
+    }
+    if (ret == 0) {
+#ifdef _WIN32
```

Comment:
I think you can just use `NO_WOLFSSL_DIR`

Reply:
Hmm, as opposed to _WIN32? The intent here is that we expect failure if on Windows, as we don't support this new function there, yet.